shared/base/router.js

```diff
@@ -166,7 +168,21 @@ _.extend(BaseRouter.prototype, Backbone.Events, {
             callback.apply(this, arguments);
           }
         }
-        realAction.call(this, params, next);
+        // in AMD environment realAction is the string containing path to the controller
+        // which will be loaded async (might be preloaded)
+        // Only used in AMD environment
+        if (isAMDEnvironment) {
+          requireAMD([realAction], function(controller) {
```

Comment:
Looks like `realAction` if a function (object) and `requireAMD` expects list of strings as dependencies.